www/source/docs/using-builder.html.md.erb

```diff
@@ -236,59 +360,248 @@ Initialize the setting from the command line with:
+
+View origin invitations:
 
 ```bash
-hab origin secret upload AWS_ACCESS_KEY_ID your-key-id
-hab origin secret upload AWS_SECRET_ACCESS_KEY your-secret-access-key
+hab origin invitations list
 ```
 
-Once your secret has been uploaded, you can refer to it in your plan file as an environment variable, and Builder will supply its decrypted value during your build job.
+Accept origin invitations:
+
+```bash
+hab origin invitations accept <ORIGIN> <INVITATION_ID>
+```
 
-For instance, if a plan required access to a file kept in a private bucket on Amazon S3, you might use the AWS CLI provided by the `core/awscli` package to download the file using secrets named `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`, which would allow the `aws` binary to read them as environment variables at build time:
+Ignore origin invitations:
 
 ```bash
-...
-pkg_build_deps=(core/awscli)
+hab origin invitations ignore <ORIGIN> <INVITATION_ID>
+```
+
+All origin members can send origin membership invitations:
 
-do_download() {
-  # When present, the AWS CLI will use the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
-  # environment variables to authenticate with Amazon S3.
-  aws s3 cp s3://your-private-bucket/your-file.tar.gz .
-}
-...
+```bash
+hab origin invitations send <ORIGIN> <INVITEE_ACCOUNT>
 ```
 
-You can delete a secret either with the Chef Habitat CLI:
+Origin owners can see all pending origin membership invitations:
 
 ```bash
-hab origin secret delete AWS_ACCESS_KEY_ID
-hab origin secret delete AWS_SECRET_ACCESS_KEY
+hab origin invitations pending <ORIGIN>
 ```
 
-or in the Builder web interface, under the Settings tab for your origin:
+Origin owners can rescind an origin membership invitation:
 
-![Builder origin secrets](/images/screenshots/origin-secrets.png)
 
-> Secrets are encrypted locally using an origin encryption key. Their values are readable only by Builder.
+```bash
+hab origin invitations rescind <ORIGIN> <INVITATION_ID>
+```
+
+Origin owners can transfer origin ownership to another member:
+
+```bash
+hab origin transfer [OPTIONS] <ORIGIN> <NEW_OWNER_ACCOUNT>
+```
+
+### Manage Member Roles with `hab rbac`
+
+You can use role based access control (RBAC) from the command line.
+An origin `MEMBER_ACCOUNT` is the name used to signin to Chef Habitat builder. You can find the list of user names on an origin's _Members Tab_. (Builder > Origin > Members)
+
+The RBAC command syntax is:
+
+```bash
+hab origin rbac <SUBCOMMAND>
+```
+
+The syntax for the `show` subcommand is:
+
+```bash
+hab origin rbac show <MEMBER_ACCOUNT> --origin <ORIGIN>
+```
+
+See an origin member's RBAC role:
+
+```
+hab origin rbac show bluewhale --origin two-tier-app
+```
+
+The syntax for the `set` subcommand is:
+
+```bash
+hab origin rbac set [FLAGS] [OPTIONS] <MEMBER_ACCOUNT> <ROLE> --origin <ORIGIN>
+```
+
+Set a member's RBAC role with:
+
+```bash
+hab origin rbac set bluewhale admin --origin two-tier-app
```

Comment:
This should be changed to `administrator`

```
<ROLE>              The role name to enforce for the member account [possible values: readonly_member, member,
                        maintainer, administrator, owner]
```

Reply:
Fixed.